templates/components/api-server/component.js

```diff
@@ -16,10 +16,9 @@ template.package = {
     "loopback": "^2.22.0",
     "loopback-boot": "^2.6.5",
     "loopback-datasource-juggler": "^2.39.0",
-    "serve-favicon": "^2.0.1"
-  },
-  "optionalDependencies": {
-    "loopback-explorer": "^1.1.0"
+    "loopback-component-explorer": "^2.1.0",
+    "serve-favicon": "^2.0.1",
+    "serve-static": "^1.10.0"
```

Comment:
> Yes i know, but in the example app these new lines were added to it, shouldn't we also add these here?

No, we shouldn't. This template is generating an "API server", the example application is built on top of this template (but it's not the same!).

In particular, the static client is configured here: https://github.com/strongloop/generator-loopback/blob/01a38d519e4b3c8b63c10a3a84c43c1b24aefdaa/example/index.js#L401-L421